### ISSUE TITLE
Not uniform value for GPS coordinates

### ISSUE THREAD

**Issue by zdila** [mattiasw/ExifReader#28]
For some pictures (like http://www.freemap.sk:3000/gallery/pictures/67129/image) the returned value of eg. `GPSLatitude` is

```json
{"value":[21,13,24.539994984954866],"description":21.223483331940262}
```

but for others (like https://user-images.githubusercontent.com/26544966/30610062-66d59c6e-9d7d-11e7-8a7b-99a26d36a8ff.JPG) it is

```json
{"value":"17,18.441667E","attributes":{},"description":"17,18.441667E"}
```

Could it be unified (in `description`)?

**Comment by mattiasw** [comment]
This is the same root problem as in #27. The value from Exif is overwritten by the XMP value. And since XMP properties can have any name, no conversions are made for the description as is done for Exif. (This should definitely be made clear in the README.) That being said, some are probably more "standardized" than others and could be converted (e.g. GPS properties and orientation). I will look into this after we have solved #27.

**Comment by disarticulate** [comment]
Within the module I'm building, I'm using isNaN(parseFloat(suspect.description)) to make sure the coordinates are legit.

I posted in #27 the fork I put together to bypass the flattening of the different tags.

**Comment by mattiasw** [comment]
Fixed in #32.
